priv/repo/migrations/20250819143942_create_api_keys.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Trento.Repo.Migrations.CreateApiKeys do
+  use Ecto.Migration
+
+  def change do
+    create table(:api_keys) do
+      add :name, :string
+      add :expire_at, :utc_datetime_usec
+      add :user_id, references(:users, on_delete: :delete_all), null: false
+
```

Comment:
Correct me if I'm wrong here, but it seems we are using the `name` field in the subsequent operations (retrieve, delete, etc). Maybe we could decouple it and just add an identifier as well? I mean, names should be unique as well, but it might be easier to use `id` rather than `name` in other endpoints.